### DISCUSSION TITLE
What are the dimensions of the Jacobian matrix?

### DISCUSSION THREAD

**Discussion by LHT-Liu** [stack-of-tasks/pinocchio#2238] [Q&A]
I calculated the Jacobian matrix of a joint on the machine. This joint should be the 11th joint, but the Jacobian matrix I got still has 15 columns (a total of 15 joints), but the values of the last four columns are 0. Why? What you get is not a matrix with 11 columns?
![image](https://github.com/stack-of-tasks/pinocchio/assets/64388319/5f8caa47-4d2c-47d2-8b91-bb4bd4d93114)

thanks ，help me
@stephane-caron

**Comment by jcarpent** [comment]
Could you share your robot mode and the Pinocchio code you use?

**Comment by jviereck** [comment]
Hi @LIUHAITAO-CH, why did you ping me on this issue?  Unless there is a specific reason for me to look into this, please avoid adding me to issues like these.

**Reply by LHT-Liu** [reply]
ok ,i'm sorry
